### PR TITLE
[BitSail][Improve][Doc] Add flow control parameters and instructions …

### PR DESCRIPTION
**Pull request by Fnck**
…#398

Signed-off-by:

## Pre-Checklist

Note: Please complete **_ALL_** items in the following checklist.

- [x] I have read through the [CONTRIBUTING.md](https://github.com/bytedance/bitsail/blob/master/docs/contributing.md) documentation.
- [x] My code has the necessary comments and documentation (if needed).
- [x] I have added relevant tests.

## Purpose

Fix of [issue 398](https://github.com/bytedance/bitsail/issues/398)

## Approaches

Add necessary text in the config markdown file

## Related Issues

N/A

## New Behavior (screenshots if needed)

N/A

### CODE REVIEW

File: website/en/documents/start/config.md [hunk 1]
```diff
@@ -81,6 +81,15 @@ Dirty record setting:(Only in batch mode)
 | dirty_records_count_threshold     | FALSE    | -1      | The threshold of the total dirty records, if dirty records bigger than the threshold, job will fail in final                      | -1      |
 | dirty_record_percentage_threshold | FALSE    | -1      | The percent threshold of the total dirty records, if dirty records percent bigger than the threshold,     job will fail in final. | -1      |
 
+Flow control setting:
+
+| Parameter name                         | Required  | Default | Description                 | Example  |
+|-----------------------------------|-------|-----|--------------------------------------------|-----|
+| reader_transport_channel_speed_byte  | FALSE | -1  | This param controls the traffic of a single concurrent reading, X bytes per second | 10  |
+| reader_transport_channel_speed_record | FALSE | -1  | This param controls the speed of a single concurrent reading, X rows per second | 10  |
+| writer_transport_channel_speed_byte | FALSE | -1  | This param controls the traffic of a single concurrent writing, Y bytes per second | 10  |
```
Comment (lichang-bd) [inline website/en/documents/start/config.md:90]: Pay attention to the alignment

File: website/zh/documents/start/config.md [hunk 2]
```diff
@@ -82,6 +82,15 @@ order: 3
 | dirty_records_count_threshold     | FALSE | -1  | 整体脏数据的阈值设置，如果在传输结束后发现脏数据多于该设置，作业失败。        | -1  |
 | dirty_record_percentage_threshold | FALSE | -1  | 整体脏数据占整体传输数据的比例，如果传输结束后发现脏数据的比例大于该阈值，作业失败。 | -1  |
 
+流控配置： 任务的流控配置
+
+| 参数名称                            | 是否必须  | 默认值 | 参数含义                                        | 示例  |
+|-----------------------------------|-------|-----|--------------------------------------------|-----|
+| reader_transport_channel_speed_byte  | FALSE | -1  | 控制单并发读的流量, X字节/S | 10  |
+| reader_transport_channel_speed_record | FALSE | -1  | 控制单并发读的速度, X条/S | 10  |
+| writer_transport_channel_speed_byte | FALSE | -1  | 控制单并发写的流量, Y字节/S | 10  |
```
Comment (lichang-bd) [inline website/zh/documents/start/config.md:91]: Pay attention to the alignment

File: website/en/documents/start/config.md (hunk 1 above)
Comment (Fnck) [reply website/en/documents/start/config.md]: Sure, change updated.

File: website/zh/documents/start/config.md (hunk 2 above)
Comment (Fnck) [reply website/zh/documents/start/config.md]: Sure, change updated.

**Review by lichang-bd — APPROVED** [review]

**Pull request merged**